pages/wiki/docserver.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Local containerized documentation server
+layout: documentation
+---
+
+This web site can be built and run within a software container, such as those used *container engines* such as [Docker](https://www.docker.com/) or [podman](https://podman.io/). One reason for doing this is to be able to install and build the documentation without having to load the collection of tools on your real computer. Building and running a documentation server in a local container is the topic of this page.
```

Comment:
`such as those used *container engines* such as`
->
`such as those created by *container engines* like` ?

Reply:
Better.  I'll make that change.